sos/report/plugins/libvirt.py

```diff
@@ -6,16 +6,16 @@
 #
 # See the LICENSE file in the source distribution for further information.
 
-from sos.report.plugins import Plugin, IndependentPlugin
+from sos.report.plugins import Plugin, RedHatPlugin
```

Comment:
NAK, we need this for UbuntuPlugin too, as we also need this from a Ubuntu perspective. Maybe, worth adding UbuntuLibvirt as a function

Reply:
ack!